scipy/stats/_stats_py.py

```diff
@@ -7549,7 +7549,8 @@ def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
                             lambda_="pearson")
 
 
-KstestResult = namedtuple('KstestResult', ('statistic', 'pvalue'))
+KstestResult = _make_tuple_bunch(['statistic', 'pvalue'],
```

Comment:
```suggestion
KstestResult = _make_tuple_bunch('KstestResult', ['statistic', 'pvalue'],
```
I think this is causing most of the test failures, if not all.